### DISCUSSION TITLE
Ian Wold | Book Club 2/2024: Recovering from TDD and Unit Tests

### DISCUSSION THREAD

**Discussion by giscus** [IanWold/ianwold.github.io#11] [Posts]
# Ian Wold | Book Club 2/2024: Recovering from TDD and Unit Tests

https://ian.wold.guru/Posts/book_club-2-2024.html

**Comment by andrewwillette** [comment]
Unit tests live rent free in your head :)

**Reply by IanWold** [reply]
Yet another injustice they have committed against me
